doc/source/configuration/monitoring.rst

```diff
@@ -169,3 +169,38 @@ If you notice ``HaproxyServerDown`` or ``HaproxyBackendDown`` prometheus
 alerts after deployment it's likely the os_exporter secrets have not been
 set correctly, double check you have entered the correct authentication
 information appropiate to your cloud and re-deploy.
+
+Friendly Network Names
+=======================
+For operators that prefer to see descriptive or friendly interface names the 
+following play can be run. This takes network names as defined in kayobe and 
+relabels the devices/interfaces in Prometheus to make use of these names.
```

Comment:
Please could we add a warning note here for anyone thinking about enabling it? Something like:

- The change creates new time series by changing labels. This will mean that for a period equal to the retention time, graphs may appear disjointed.
- Any custom Grafana or Prometheus configuration added by the customer may be affected by the relabelling
- Billing systems monitoring network traffic may need to be updated  
- The feature currently requires that network interface names are consistent across each network
- Before turning on the feature, it should be discussed with the customer. All points above must be considered.